qdrant/v1.1.x/comparisons.md

```diff
@@ -0,0 +1,69 @@
+|----------------|----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Index**      | **Collection** | Pinecone's index is an organizational unit for storing and managing vectors of the same size. The index is tightly coupled with hardware (pods). Qdrant uses the collection to describe a similar concept, however, a single instance may handle multiple collections at once. |
+| **Collection** | N/A            | A collection in Pinecone is a static copy of an *index* that you cannot query, mostly used as some sort of backup. There is no such concept in Qdrant, but if you want to back your collection up, you may always create a snapshot.                                           |
+| **Namespace**  | N/A            | Namespaces allow the partitioning of the vectors in an index into subsets. Qdrant does not have such a concept, as you can always create multiple collections.                                                                                                                 |
+| **Metadata**   | **Payload**    | Additional attributes describing a particular object, other than the embedding vector. Both engines support various data types, but Pinecone metadata is key-value, while Qdrant supports any JSON-like objects.                                                               |
+| **Query**      | **Search**     | Name of the method used to find the nearest neighbors for a given vector, possibly with some additional filters applied on top.                                                                                                                                                |
+| N/A            | **Scroll**     | Pinecone does not offer a way to iterate through all the vectors in a particular index. Qdrant has a `scroll` method to get them all without using search.                                                                                                                     |
+
+#### Known metadata limitations
+
+1. Pinecone does not support any JSON metadata, but a flat structure with strings, numbers, booleans, or lists of strings used as values. Qdrant accepts any JSON object as a payload, even nested structures.
+2. NULL values are not supported in Pinecone metadata but are handled properly by Qdrant.
+3. The maximum size of Pinecone metadata is 40kb per vector. 
+4. Pinecone, unlike Qdrant, does not support geolocation and filtering based on geographical criteria.
+
+It is worth mentioning, that **Pinecone will automatically create metadata indexes for all the fields**. Qdrant assumes you know
+your data and your future queries best, so it's up to you to choose the fields to be indexed. Thus, **you need to explicitly 
+define the payload indexes while using Qdrant**.
+
```

Comment:
After this metadata section, do we want to mention some practical scenarios where does each tool shine? 

```suggestion
Here are scenarios and use cases where each of these vector search engines, Qdrant and Pinecone, could be particularly advantageous:

**Qdrant**

1. **Complex Filtering Operations**: If your use case involves complex filtering operations, especially at a large scale, Qdrant's extended filtering support might make it the more advantageous choice. This could be particularly beneficial for applications such as neural network or semantic-based matching and faceted search. Note that Qdrant supports Geospatial queries as well. 

2. **Data Privacy and Control**: For organizations in industries with data privacy regulations or those that prioritize complete control over their data and infrastructure, Qdrant's self-hosted solution could be a strong choice. 

3. **Power User Needs**: Built in the *Nix philosophy of doing one thing well, Qdrant is targeted towards developers looking for a performant, excellent vector search engine — but they've to do the rest e.g. query caching, creating embeddings, telemetry for their apps on their own. 

**Pinecone**

1. **Web-based Vector Search Applications**: Pinecone shines as a cloud-native vector database with a simple API, making it a strong choice for use cases that are just getting started. 

2. **Live Index Updates**: If your project involves adding, editing, or deleting data frequently and you need your index to be updated in real-time, Pinecone's simple APIs live index updates could make it a more advantageous choice.

The choice between Qdrant and Pinecone should be determined by the specific requirements of your project, including factors such as the scale of your data, the complexity of your use case, your budget, and what trade offs do you care about the most.


```